api/src/main/java/com/github/eyefloaters/console/api/model/Topic.java

```diff
@@ -277,6 +279,17 @@ public Long getRecordCount() {
             .orElse(null);
     }
 
+    public BigInteger getSize() {
```

Comment:
We should document the semantics of this.

Reply:
Added a description to `getSize` on the topic to indicate that it is derived as the total of all partition leader's local storage size. I also added a comment to the description that it may include remote size(s) when tiered storage is supported by the API.

Reply:
I wonder if we should rename it too. `Size` is ambiguous in multiple dimensions (it is bytes or records, does it include replication or not?). `TotalLeaderLogBytes` is a bit longer, but it's completely clear what it is.

Incidentally, we should consider adopting the same convention Kafka uses for quantities represented using numeric types: Always include the unit in the name of the variable, field or method. This makes it much easier when reading code and is especially helpful for times where both seconds and milliseconds are common. Messing up units is such an easy mistake to make.

Reply:
Renamed to `TotalLeaderLogBytes` :+1: